### DISCUSSION TITLE
warning for self-signed certificate for local instance

### DISCUSSION THREAD

**Discussion by vicjoe** [nextcloud/all-in-one#1519] [Questions] [answered]
Because could not open the ports to get certificate via ACME , so set the self-signed cerificate and reverse proxy, seems all the containers are up , and the major photo function is runing smooth.
![image](https://user-images.githubusercontent.com/31268451/207229230-15ea2350-625b-40e5-87de-b28dc934c479.png)
However, some log warnings below for the cerficate , 
![image](https://user-images.githubusercontent.com/31268451/207230480-adf792ea-f2fb-45b9-b97c-123582ac7bbb.png)

1- Not sure which part of the function will be impacted,  only notification push or collabora  ?
any risk if only to run the server as normal document cloud ? 
2- if purchase a formal certicate, where is the directory to store to boot the nextcloud perfectly ?
Thanks ~

**Comment by szaimen** [comment]
Hi, see https://github.com/nextcloud/all-in-one/discussions/1155#discussioncomment-3660182

**Comment by szaimen** [comment, marked as answer]
> 1- Not sure which part of the function will be impacted, only notification push or collabora ?
any risk if only to run the server as normal document cloud ?

Sorry, I don't know as I haven't tested it with self-signed certificates as they are not supported anyway.

> 2- if purchase a formal certicate, where is the directory to store to boot the nextcloud perfectly ?

You need to to specify the bought certificate to be used by your reverse proxy. This should allow Nextcloud to work correcrtly, yes
